### ISSUE TITLE
Single-strain community

### ISSUE THREAD

**Issue by bhillmann** [knights-lab/analysis_SHOGUN#11]
- [ ] Simulate reads from a single genome and taxonomically profile it using various tools
- [x] Find an assembly sequencing run for a single strain and taxonomically profile it using various tools

**Comment by RRShieldsCutler** [comment]
By "single strain in silico community" do you mean a "community" of manufactured shotgun data derived from known bugs, where there's only one strain per ... species? genus?

**Comment by bhillmann** [comment]
No, I mean using a read simulation tool such as [dwgsim](https://github.com/nh13/dwgsim) to simulate reads from a single organism. Community is the wrong term here.

**Comment by RRShieldsCutler** [comment]
This group [sequenced a few E.coli](https://bmcgenomics.biomedcentral.com/articles/10.1186/1471-2164-14-675) for assembly on MiSeq 2x150, nextera kit.

The paired reads for Bal225 are in analysis_SHOGUN/single-species/ecoli_Bal225/
QC processed reads (both SE and stitched) and downsampled reads are in shi7en_161201/

**Comment by RRShieldsCutler** [comment]
Shogun analysis on wgs for Escherichia coli Bal225.
R1.fna "deep" = 1,647,613 reads
R1.fna "deep" = 1,689,767 reads
_stitched.fna "deep" = 559970 reads (not that deep)
All downsampled stuff = 500,000 reads

There is no strain level information in the utree_abfvh database, so it's just hitting E.coli;t__
[deep_taxon_counts.txt](https://github.com/knights-lab/analysis_SHOGUN/files/628049/deep_taxon_counts.txt)
[shallow_taxon_counts.txt](https://github.com/knights-lab/analysis_SHOGUN/files/628050/shallow_taxon_counts.txt)

**Comment by RRShieldsCutler** [comment]
Update: Using the new utree binaries... more super-low count hits outside E.coli, otherwise similar.
[deep_taxon_counts_new-utree.txt](https://github.com/knights-lab/analysis_SHOGUN/files/628212/deep_taxon_counts_new-utree.txt)
[shallow_taxon_counts_new-utree.txt](https://github.com/knights-lab/analysis_SHOGUN/files/628213/shallow_taxon_counts_new-utree.txt)

**Comment by RRShieldsCutler** [comment]
And similar analysis with a K. pneumoniae run from [this paper](http://jac.oxfordjournals.org/content/68/10/2234.full).
Deep R1 = 2,800,699 reads
Deep R2 = 2,663,982 reads

[deep_taxon_counts_kpneumo.txt](https://github.com/knights-lab/analysis_SHOGUN/files/628475/deep_taxon_counts_kpneumo.txt)
[shallow_taxon_counts_kpneumo.txt](https://github.com/knights-lab/analysis_SHOGUN/files/628476/shallow_taxon_counts_kpneumo.txt)

**Comment by RRShieldsCutler** [comment]
This dataset:
http://www.nature.com/nature/journal/v498/n7452/full/nature12198.html

**Comment by RRShieldsCutler** [comment]
And yet another for S. aureus from [this paper](http://jcm.asm.org/content/52/4/1182.full). This one worked well!
Deep R1 = 7,877,713 reads
Deep R2 = 7,944,811 reads

[staph_deep_taxon_counts.txt](https://github.com/knights-lab/analysis_SHOGUN/files/631766/staph_deep_taxon_counts.txt)
[staph_shallow_taxon_counts.txt](https://github.com/knights-lab/analysis_SHOGUN/files/631767/staph_shallow_taxon_counts.txt)

**Comment by bhillmann** [comment]
Are you running these with the newest version of UTree? I pushed a new version to my bin folder that includes confidence scores for assignment.

**Comment by bhillmann** [comment]
I created a RefSeq complete dataset, that is only NC prefices, can we re-run this with SHOGUN and see if the number of false positives drops?

```
/project/flatiron2/data/refseq_abfhv/complete/utree
```

DB stats:

Number of Archaea: 287
```
grep "k__Archaea" ./refseq.abfvh.all.annotated.map | wc -l
```
Number of Bacteria: 7001
```
 grep "k__Bacteria" ./refseq.abfvh.all.annotated.map | wc -l
```
Number of Eukaryota: 811 - Number of Human
```
"k__Euk" ./refseq.abfvh.all.annotated.map | wc -l
```
Number of Human: 194
```
grep "Homo" ./refseq.abfvh.all.annotated.map | wc -l
```
Number of Viruses: 7239
```
grep "k__Viruses" ./refseq.abfvh.all.annotated.map | wc -l
```
Number of Sequences:
```
grep "^>" ./refseq.abfvh.all.annotated.fna | wc -l
```
15389

Number of BPs:
```
cat ./refseq.abfvh.all.annotated.fna | paste - - | cut -f2 | wc -c
```
13287499117

**Comment by bhillmann** [comment]
This can be done tomorrow.

**Comment by RRShieldsCutler** [comment]
This is done. Data are in:
```
/project/flatiron2/ben/projects/analysis_SHOGUN/single_strain/results/ecoli_shogun_161208
/project/flatiron2/ben/projects/analysis_SHOGUN/single_strain/results/kleb_shogun_161208
/project/flatiron2/ben/projects/analysis_SHOGUN/single_strain/results/staph_shogun_161208
```
Run looks like:
```
shogun_utree_lca -i /project/flatiron2/ben/projects/analysis_SHOGUN/single_strain/data/s_aureus/shi7en_161205/truseq3-2/fasta -u /project/flatiron2/data/refseq_abfhv/complete/utree/refseq.abfvh.all.ctr -p 12
```

**Comment by RRShieldsCutler** [comment]
Finished rerunning with the new utree (complete, species). Will take a look at the data, but it's in the same directories as above, noted with "species" (e.g. "ecoli_shogun-species_161208").
